src/Tooling/Bit.Tooling.Templates/TodoTemplate/Api/Controllers/AttachmentController.cs

```diff
@@ -1,26 +1,20 @@
-﻿using MimeTypes;
+﻿using Bit.Tooling.SourceGenerators;
+using MimeTypes;
 using TodoTemplate.Api.Models.Account;
 using SystemFile = System.IO.File;
 
 namespace TodoTemplate.Api.Controllers;
 
 [Route("api/[controller]")]
 [ApiController]
-public class AttachmentController : ControllerBase
+public partial class AttachmentController : ControllerBase
 {
-    private readonly AppSettings _appSettings;
-    private readonly UserManager<User> _userManager;
-    private readonly IWebHostEnvironment _webHostEnvironment;
-
-    public AttachmentController(IOptionsSnapshot<AppSettings> setting,
-        UserManager<User> userManager,
-        IWebHostEnvironment webHostEnvironment)
-    {
-        _appSettings = setting.Value;
-        _userManager = userManager;
-        _webHostEnvironment = webHostEnvironment;
-    }
+    [AutoInject] private readonly AppSettings _appSettings;
```

Comment:
Have you tested this? @mohabbati 
Use public properties with pascal naming

Reply:
I tested it. It works well.
I'll change all fields to property.
@ysmoradi